integration/test/basic/project.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1
+
+test:
+  env:
+    - "KEEP_RESOURCES=${KEEP_RESOURCES}"
+    - "CIRCLE_SHA1=${CIRCLE_SHA1}"
```

Comment:
I think this can actually be removed. Can somebody confirm?

Reply:
Exactly, same as above, only needed for CNR-related stuff

Reply:
Sweet. Removed.